common/djangoapps/student/models.py

```diff
@@ -1038,7 +1038,7 @@ def num_enrolled_in(self, course_id):
         enrollment_number = super(CourseEnrollmentManager, self).get_queryset().filter(
             course_id=course_id,
             is_active=1
-        ).count()
+        )[:max_enrollment_for_buttons + 1].count()
```

Comment:
this should also be changed to max_enrollments

Reply:
My bad.